data_store/controllers/load_functions.py

```diff
@@ -203,7 +204,7 @@ def get_submission_by_programme_and_round(
 
 def get_or_generate_submission_id(
     programme_exists_same_round: Programme | None, round_number: int, fund_code: str
-) -> tuple[str, Submission | None]:
+) -> tuple[str, GUID | None]:
```

Comment:
oh nice it actually made us fix something that was wrong 👏

Reply:
But at what cost, Sam! AT WHAT COST?!